cmd/clef/main.go

```diff
@@ -545,12 +545,12 @@ func signer(c *cli.Context) error {
 		if err != nil {
 			utils.Fatalf("Could not start RPC api: %v", err)
 		}
-		extapiURL = fmt.Sprintf("http://%s", httpEndpoint)
+		extapiURL = fmt.Sprintf("http://%s", listener.Addr())
```

Comment:
should this be `listener.Addr().String()` ?

Reply:
Hm, it seems to work, I wonder why, because you're right. I'll swap it to `%v`, which is a better default.